File: 220818-weridString/weirdString.test.js
import { stringTransformer, solution } from './weirdString';

test('transform', () => {
  expect(stringTransformer('try')).toBe('TrY');
  expect(stringTransformer('hello')).toBe('HeLlO');
  expect(stringTransformer('world')).toBe('WoRlD');
});

test('solution', () => {
  expect(solution('try hello world')).toBe('TrY HeLlO WoRlD');
});

File: 220818-weridString/app/src/test/java/WeirdStringTest.java
import org.junit.jupiter.api.*;

import static org.junit.jupiter.api.Assertions.*;

class WeirdStringTest {
  @Test
  void nullTest() {
   WeirdString weirdString = new WeirdString();

    assertEquals("TrY",weirdString.solution("try"));
    assertEquals("HeLlO",weirdString.solution("hello"));
  }

}